R/bibtex.R

```diff
@@ -215,10 +309,8 @@ do_read_bib <- function(file, encoding = "unknown", srcfile){
 #'         stats, stats4, tools and utils) are treated specially: this package
 #'         contains bibtex entries for these packages.
 #' @param encoding encoding
-#' @param header header of the citation list.
-#'        By default this is made from the \samp{Preamble} entries found in
-#'        the bib file.
-#' @param footer footer of the citation list
+#' @param header DEPRECATED.
+#' @param footer DEPRECATED
```

Comment:
Those were never used

Reply:
See my previous note on deprecating arguments. 

If a previous argument was never used, perhaps we could also just exclude these arguments altogether in the next version?

Reply:
Done. I would favor removing those on the next major version (1.0.0) just in case. For v0.5.0 I prefer to leave them, so we can get feedback